exec/java-exec/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -215,11 +223,12 @@ SqlNode SqlDropView() :
 
 /**
  * Parses a CTAS or CTTAS statement.
- * CREATE [TEMPORARY] TABLE tblname [ (field1, field2, ...) ] AS select_statement.
+ * CREATE [TEMPORARY] TABLE [IF NOT EXISTS] tblname [ (field1, field2, ...) ] AS select_statement.
  */
 SqlNode SqlCreateTable() :
 {
     SqlParserPos pos;
+    boolean tableNonExistenceCheck = false;
```

Comment:
Not critical but I am in favor of adding new parameters in the end. Of course if logically it is not required.

Reply:
Moved the new parameter to the end